examples/rhoai-quickstart/feast-demo-quickstart.ipynb

```diff
@@ -0,0 +1,1530 @@
+       "    <tr>\n",
+       "      <th>1804</th>\n",
+       "      <td>2021-04-12 07:00:00+00:00</td>\n",
+       "      <td>1001</td>\n",
+       "      <td>0.039615</td>\n",
+       "      <td>0.870356</td>\n",
+       "      <td>271</td>\n",
+       "      <td>2024-09-24 03:07:44.487</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1805</th>\n",
+       "      <td>2024-09-16 15:00:00+00:00</td>\n",
+       "      <td>1003</td>\n",
+       "      <td>0.506131</td>\n",
+       "      <td>0.342303</td>\n",
+       "      <td>799</td>\n",
+       "      <td>2024-09-24 03:07:44.487</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1806</th>\n",
+       "      <td>2024-09-16 15:00:00+00:00</td>\n",
+       "      <td>1003</td>\n",
+       "      <td>0.506131</td>\n",
+       "      <td>0.342303</td>\n",
+       "      <td>799</td>\n",
+       "      <td>2024-09-24 03:07:44.487</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>1807 rows × 6 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "               event_timestamp  driver_id  conv_rate  acc_rate  \\\n",
+       "0    2024-09-09 03:00:00+00:00       1005   0.136662  0.043697   \n",
+       "1    2024-09-09 04:00:00+00:00       1005   0.027620  0.808205   \n",
+       "2    2024-09-09 05:00:00+00:00       1005   0.523759  0.807032   \n",
+       "3    2024-09-09 06:00:00+00:00       1005   0.795781  0.674151   \n",
+       "4    2024-09-09 07:00:00+00:00       1005   0.075056  0.902140   \n",
+       "...                        ...        ...        ...       ...   \n",
+       "1802 2024-09-24 01:00:00+00:00       1001   0.163166  0.803832   \n",
+       "1803 2024-09-24 02:00:00+00:00       1001   0.102868  0.695487   \n",
+       "1804 2021-04-12 07:00:00+00:00       1001   0.039615  0.870356   \n",
+       "1805 2024-09-16 15:00:00+00:00       1003   0.506131  0.342303   \n",
+       "1806 2024-09-16 15:00:00+00:00       1003   0.506131  0.342303   \n",
+       "\n",
+       "      avg_daily_trips                 created  \n",
+       "0                  22 2024-09-24 03:07:44.487  \n",
+       "1                 953 2024-09-24 03:07:44.487  \n",
+       "2                 739 2024-09-24 03:07:44.487  \n",
+       "3                 743 2024-09-24 03:07:44.487  \n",
+       "4                 724 2024-09-24 03:07:44.487  \n",
+       "...               ...                     ...  \n",
+       "1802              648 2024-09-24 03:07:44.487  \n",
+       "1803              361 2024-09-24 03:07:44.487  \n",
+       "1804              271 2024-09-24 03:07:44.487  \n",
+       "1805              799 2024-09-24 03:07:44.487  \n",
+       "1806              799 2024-09-24 03:07:44.487  \n",
+       "\n",
+       "[1807 rows x 6 columns]"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
```

Comment:
You should introduce this step clarifying that this is the static file containing historical data (randomly generated by `feast init` command) as defined by the `FileSource` created in the example repo:
```py
driver_stats_source = FileSource(
    name="driver_hourly_stats_source",
    path="_YOUR_PATH_/feature_repo/data/driver_stats.parquet",
    timestamp_field="event_timestamp",
    created_timestamp_column="created",
)
```

Reply:
I have added more details to the description. Please let me know if it is not very clear.